projects/is-cron-editor/src/lib/is-cron-editor.component.ts

```diff
@@ -0,0 +1,737 @@
+import {Component, Input, OnInit, Output} from '@angular/core';
+import {ControlValueAccessor, FormControl, NG_VALUE_ACCESSOR} from '@angular/forms';
+import {Subscription} from 'rxjs';
+import {debounceTime} from 'rxjs/operators';
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: 'is-cron-editor',
+  templateUrl: './is-cron-editor.component.html',
+  styleUrls: ['./is-cron-editor.component.css'],
+  providers: [{
+    provide: NG_VALUE_ACCESSOR,
+    useExisting: IsCronEditorComponent,
+    multi: true
+  }]
```

Comment:
Also please implement `Validator` interface. Angular forms will call it and mark your control invalid in case there is invalid cron expression